packages/repack/js/index.ts

```diff
@@ -0,0 +1,6 @@
+/* 
+    TurboModuleRegistry.get taps into the old Native Modules API under the hood,
+    we need to re-export our module, to avoid registering it multiple times.
+*/
+
+export default require('./NativeScriptManager').default;
```

Comment:
Does it work for the old arch as well? 
AFAIR I did a check for that 

```
const isTurboModuleEnabled = global.__turboModuleProxy != null;
```

Reply:
Yes, this works and as confirmation I checked how modules are called on [react native svg](https://github.com/software-mansion/react-native-svg/blob/d31ffdb762d34171fd934ac97b031ccf5e0b645d/src/elements/Svg.tsx#L96)